laravel/the-basics/laravel-volume-storage.html.markerb

```diff
@@ -96,6 +96,8 @@ It's the default burrow for session, cache, and file data amongst others. If you
 
 <div class="callout">
 Fly Volumes do not automatically sync their data with each other. Please remember to create the appropriate data-replication logic if your Fly App will be using more than one volume instance, and if your app requires data available across its Volumes.
+
+Of course there's a bunch of workarounds! Have you heard about Tigris? It's an S3-compatible, globally caching, object storage that can manage your buckets and create CDN-like behavior with zero-configuration! Check out its integration with [Laravel here](/docs/laravel/the-basics/laravel-tigris-file-storage/)!
```

Comment:
Let's put this in a more positive light.  Tigris isn't just a workaround, it is an alternative with unique advantages for some use cases.  At the top of this page you mention "session, cache, and file data".   Which of those use cases are a good match for Tigris?  And perhaps mention Tigris up front so that a reader doesn't get through this page only to find out that they are on the wrong page.